html/modules/custom/reliefweb_utility/src/Helpers/LocalizationHelper.php

```diff
@@ -134,10 +133,13 @@ public static function getCollator($language = NULL) {
     $language = static::getLanguage($language);
 
     if (!isset($collators[$language])) {
-      if (function_exists('collator_create')) {
-        $collator = collator_create($language);
+      $collator = static::collatorCreate($language);
 
-        switch (intl_get_error_code()) {
+      if (!$collator) {
+        $collators[$language] = FALSE;
```

Comment:
```suggestion
        $collator = FALSE;
```

To be compatible with line 166 below: `$collators[$language] = $collator;` otherwise this will store a `NULL` if collator_create failed in `static::collatorCreate`.

Alternatively, have `static::collatorCreate` return FALSE if the collator couldn't be created and replace the the `if/else` with a `if ($collator !== FALSE) {` before the switch.